source/agg-pipeline-builder/save-agg-pipeline.txt

```diff
@@ -17,12 +17,6 @@ you load a saved pipeline, you can make edits to the pipeline stages without
 changing the original saved pipeline. You can also create a :ref:`view 
 <view-details>` from your pipeline results.
 
-Before You Begin 
```

Comment:
Removed this because you don't need to have a completed pipeline to be able to save it.